docs/user/alerting/rule-types/geo-rule-types.asciidoc

```diff
@@ -10,24 +10,90 @@
 
 The tracking containment rule alerts when an entity is contained or no longer contained within a boundary. 
 
+In *{stack-manage-app}* > *{rules-ui}*, click *Create rule*, fill in the name and optional tags, then select *Tracking containment*.
+
 [float]
-=== Requirements
-To create a tracking containment rule, the following requirements must be present:
+=== Define the conditions
+
+When you create a tracking containment rule, you must define the conditions that it detects. For example:
+
+[role="screenshot"]
+image::user/alerting/images/alert-types-tracking-containment-conditions.png[Creating a tracking containment rule in Kibana,width=75%]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+. Define the entities index
+  It must contain a `geo_point` or `geo_shape` field, `date` field, and entity identifier.
+  An entity identifier is a `keyword`, `number`, or `ip` field that identifies the entity.
+  Entity data is expected to be updating so that there are entity movements to alert upon.
+
+. Define the boundaries index, which contains `geo_shape` data.
+  Boundaries data is expected to be static (not updating).
+  Boundaries are collected once when the rule is created and anytime after when boundary configuration is modified.
 
-- *Entities index*: An index containing a `geo_point` or `geo_shape` field, `date` field, and entity identifier. An entity identifier is a `keyword`, `number`, or `ip` field that identifies the entity. Entity data is expected to be updating so that there are entity movements to alert upon.
-- *Boundaries index*: An index containing `geo_shape` data.
-Boundaries data is expected to be static (not updating). Boundaries are collected once when the rule is created and anytime after when boundary configuration is modified.
+. Set the check interval, which defines how often to evaluate the rule conditions. 
 
-Entity locations are queried to determine if they are contained within any monitored boundaries.
-Entity data should be somewhat "real time", meaning the dates of new documents aren’t older
-than the current time minus the amount of the interval. If data older than
-`now - <current interval>` is ingested, it won't trigger a rule.
+Entity locations are queried to determine whether they are contained within any monitored boundaries.
+Entity data should be somewhat "real time", meaning the dates of new documents aren't older than the current time minus the amount of the interval.
+If data older than `now - <current interval>` is ingested, it won't trigger a rule.
```

Comment:
> If data older than `now - <current interval>` is ingested, it won't trigger a rule.

Where is the "current interval" defined?

Reply:
The current interval is the user provided value from "Check every" input.

Reply:
Thanks, I've updated to use "check interval" terminology: bcdccadef4d2fd31a7625a2f11f66b02e95ccc80